pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -127,7 +127,7 @@ public static Object jsonPath(Object object, String jsonPath) {
   /**
    * Extract from Json with path to String
    */
-  @ScalarFunction
+  @ScalarFunction(nullableParameters = true)
```

Comment:
Similarly, we don't need to annotate this function, and the null check can be removed